modules/administration/pages/image-management.adoc

```diff
@@ -403,24 +404,39 @@ Disk space depends on the actual size of the image.
 For more information, see the documentation of the underlying system.
 ====
 
+[WARNING]
+====
+The build host must be a Salt client.
+Do not install the build host as a traditionally managed client.
+====
 
 
 [[at.images.kiwi.buildhost]]
 === Create a Build Host
 
 
 To build all kinds of images with {productname}, create and configure a build host.
-OS Image build hosts are Salt clients running {sles}{nbsp}12 (SP3 or later) or 15.
+OS Image build hosts are Salt clients running on {sles}{nbsp}12 (SP3 or later) or {sles}{nbsp}11 SP4.
+
 This procedure will guide you through the initial configuration for a build host.
 
-From the {productname} {webui}, perform these steps to configure a build host:
+[IMPORTANT]
+====
+The operating system on the build host must match the operating system on the targeted image.
+
+For example, build {sles}{nbsp}12 based images on a build host running {sles}{nbsp}12 OS version.
+Build {sles}{nbsp}11 based images on a build host running {sles}{nbsp}11 OS version.
+====
+
+
+Configure the build host in the {productname} {webui}:
 
 . Select a client that will be designated as a build host from the menu:Systems[Overview] page.
-. From the menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
+. Navigate to menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
```

Comment:
```suggestion
. Navigate to the menu:System Details[Properties] tab, enable the [guimenu]``Add-on System Type`` [guimenu]``OS Image Build Host``.
Confirm with btn:[Update Properties].
```
Even if a label is finished with ":", I'd not add the colon if you reference the name of the label in the running text.